cores/esp32/esp32-hal-rmt.c

```diff
@@ -23,9 +23,15 @@
 #include "driver/periph_ctrl.h"
 #include "esp_intr_alloc.h"
 
+#include "driver/rmt.h"
+
 /**
  * Internal macros
  */
+
+// USE SOC_RMT_CHANNELS_PER_GROUP
+// LOOK to https://github.com/espressif/esp-idf/blob/master/components/hal/include/hal/rmt_types.h#L32
+
```

Comment:
correct :) https://github.com/espressif/arduino-esp32/blob/master/tools/sdk/esp32/include/soc/esp32/include/soc/soc_caps.h#L195-L202
```cpp
#include "soc/soc_caps.h"

#define MAX_CHANNELS (SOC_RMT_GROUPS * SOC_RMT_CHANNELS_PER_GROUP)
#define MAX_DATA_PER_CHANNEL SOC_RMT_MEM_WORDS_PER_CHANNEL
#define MAX_DATA_PER_ITTERATION (MAX_DATA_PER_CHANNEL - 2)
```

It is then chip independent. We should try to have as few chips specific things as possible

Reply:
OK. Done.